docs/common/schedules.rst

```diff
@@ -0,0 +1,7 @@
+.. _schedules:
+
+Schedules
+=========
+
```

Comment:
Describe as used as hyperparameter for most of the algortihms, in order to change the value over time of some parameters (usual LR)